test/src/rabbit_hash_exchange_test.erl

```diff
@@ -62,10 +62,14 @@ test0(MakeMethod, MakeMsg, Queues, MsgCount, Count) ->
                                                  routing_key = <<"">> })
      || Q <- Queues],
 
+    amqp_channel:call(Chan, #'confirm.select'{}),
+
     [amqp_channel:call(Chan,
                        MakeMethod(),
                        MakeMsg()) || _ <- lists:duplicate(MsgCount, const)],
 
+    amqp_channel:wait_for_confirms_or_die(Chan),
+
```

Comment:
I was getting intermittent failures and I thought maybe there was a race between publishing the message and getting the number of messages on the queue. Does that sound plausible?